packages/ariakit/src/checkbox/__examples__/controlled-checkbox/index.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from "react";
+import { Checkbox } from "ariakit/checkbox";
+import "./style.css";
+
+export default function Example() {
+  const [checked, setChecked] = React.useState(true);
+  const toggle = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { checked } = e.target;
+    setChecked(checked);
+  };
```

Comment:
What if we pass the function directly to the `onChange` prop:
```jsx
onChange={(event) => setChecked(event.target.checked)}
```
I think we won't need the explicit type then (not sure).